swatch-core/src/main/java/org/candlepin/subscriptions/clowder/ClowderJsonPathPropertySource.java

```diff
@@ -135,30 +126,23 @@ public ClowderJsonPathPropertySource() throws IOException {
 
   @Override
   public Object getProperty(String name) {
-    if (!name.startsWith(PREFIX)) {
+    if (!name.startsWith(CLOWDER)) {
       return null;
     }
 
-    // handling for special properties like kafka brokers
-    if (name.contains(KAFKA_BROKERS)) {
-      String brokerConfig = name.substring(PREFIX.length() + KAFKA_BROKERS.length());
-      // special logic to provide a comma-separated list of kafka brokers
-      // value is expected to be a list of key-value collection with the broker configuration.
-      KafkaBrokerConfigMapper configFunction = KAFKA_BROKERS_PROPERTIES.get(brokerConfig);
-      if (configFunction != null) {
-        return getKafkaBrokerConfig(name, configFunction);
+    for (var strategy : handledPropertyStrategies.entrySet()) {
+      if (name.contains(strategy.getKey())) {
```

Comment:
How would you feel about altering this to strip the `clowder.` off the name and then use `startsWith`.  I'm a little nervous that we might someday end up with something like `clowder.kafka-brokers.topics.endpoints` or something along those lines that would end up resolving to the `endpoints` path instead of the `kafka-brokers` path.  Plus, if we expand the map of strategies in the future, that increases the odds of this type of ambiguity.

Reply:
Yes, this makes totally sense +1